src/credentialstore/linux/file-token-storage.ts

```diff
@@ -52,12 +52,12 @@ export class FileTokenStorage {
         return deferred.promise;
     }
 
-    public RemoveEntries(entriesToRemove: Array<any>, entriesToKeep: Array<any>) : Q.Promise<void> {
+    public RemoveEntries(entriesToKeep: Array<any> /*, entriesToRemove?: Array<any>*/) : Q.Promise<void> {
```

Comment:
Yes, in case I ever actually want RemoveEntries to, well, remove the entries that are passed.  :-)